src/main/java/org/cyclonedx/gradle/CycloneDxTask.java

```diff
@@ -104,7 +100,8 @@ public class CycloneDxTask extends DefaultTask {
     private final ListProperty<String> skipConfigs;
     private final ListProperty<String> skipProjects;
     private final Property<File> destination;
-
+    private final Property<OrganizationalEntity> organizationalEntity;
```

Comment:
To avoid serialization issue, you can remove the `Property`. That should simplify your code. 

```suggestion
    private OrganizationalEntity organizationalEntity;
```

Reply:
Even without Property it is not possible to define an @Input getOrganizationalEntity() since OrganizationalEntity is not serialziable.